commands/duck.js

```diff
@@ -0,0 +1,28 @@
+const got = require('got');
+
+module.exports = {
+	name: 'dog',
+	ping: true,
+	description: 'This command will give you a link to a picture of a random duck',
+	permission: 100,
+	category: 'Random command',
+	execute: async (channel, user, input, perm) => {
+		try {
+			if (module.exports.permission > perm) {
+				return;
+			}
+			const image = await got('https://random-d.uk/api/v2/random', { timeout: 10000 }).json();
+
+			return `DuckerZ quack ${image.message}`;
```

Comment:
message should be changed to url

Reply:
Fixed in 4ecaa183ede7d7ffc730ff4b5c1b08666f0d613d